java/system-test/system-test-drivers/src/test/java/sleeper/systemtest/drivers/util/WaitForJobsStatusTest.java

```diff
@@ -103,11 +103,45 @@ void shouldReportSeveralCompactionJobs() {
                 "  },\n" +
                 "  \"numUnstarted\": 1,\n" +
                 "  \"numUnfinished\": 2,\n" +
-                "  \"firstInProgressStartTime\": \"2023-09-18T14:48:01Z\",\n" +
-                "  \"longestInProgressDuration\": \"PT2M\"\n" +
+                "  \"firstInProgressStartTime\": \"2023-09-18T14:48:00Z\",\n" +
+                "  \"longestInProgressDuration\": \"PT2M1S\"\n" +
```

Comment:
This seems incorrect - this is the time the finished job started. It's not in progress any more.